app/client/components/sectionHeader.tsx

```diff
@@ -13,6 +13,7 @@ import { gridBase, gridItemPlacement } from "../styles/grid";
 
 interface SectionHeaderProps {
   title: string | JSX.Element;
+  isArticlePage?: boolean;
```

Comment:
From a meeting the team just had I realised that it might be better for now the `pageHasNav` as prop because of how we determine which topic we're on from the Article.